### ISSUE TITLE
Hub access is broken by PR1219. 

### ISSUE THREAD

**Issue by ailzhang** [pytorch/fairseq#1357]
https://github.com/pytorch/fairseq/pull/1219 broke hub. 
Repro: 
`en2fr = torch.hub.load('pytorch/fairseq', 'transformer.wmt14.en-fr', tokenizer='moses', bpe='subword_nmt')`
Error message:
```
---> 27             from subword_nmt.subword_nmt import apply_bpe
     28             bpe_parser = apply_bpe.create_parser()

ModuleNotFoundError: No module named 'subword_nmt'

During handling of the above exception, another exception occurred:
```
Also would you mind adding hub command into CI if you have one? ;D Thanks!

**Comment by myleott** [comment]
Thanks for flagging, I'll submit a fix shortly and add some tests for hub commands.

**Comment by myleott** [comment]
Fixed in e404785250d0b96c1e6ef2213e29ada1a97b073b
